renovate.json5

```diff
@@ -1,6 +1,15 @@
 {
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",

```

Comment:
This diff is not related about pull-request but it enable completion in VSCode with yaml extensions, so I recommend to add it.